src/modules/cmdpal/Exts/Microsoft.CmdPal.Ext.WindowWalker/Commands/ExplorerInfoResultCommand.cs

```diff
@@ -0,0 +1,67 @@
+﻿// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Diagnostics;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.CmdPal.Ext.WindowWalker.Components;
+using Microsoft.CmdPal.Extensions;
+using Microsoft.CmdPal.Extensions.Helpers;
+
+namespace Microsoft.CmdPal.Ext.WindowWalker.Commands;
+
+internal sealed partial class ExplorerInfoResultCommand : InvokableCommand
```

Comment:
Sorry i'm being daft - where is this used? what does this do? To me it seems weird that we'd need to run something as another user... to switch to that hwnd...?

Reply:
Much of this is lift and shifted from existing PT code. 

This is here almost as an informational list item to the user that you can't actually kill explorer processes because... well I think the OS doesn't like it at all... but what do I know.

![image](https://github.com/user-attachments/assets/163f56cf-ed31-45e2-ac9b-e0513b5ef043)

This is what it looks like in PT Run today. 

It only shows up with specific settings also toggled:

![image](https://github.com/user-attachments/assets/5ea021ac-817a-4498-9d6d-7b089c73e25c)

That one has to be untoggled. But... wait we don't have the ability to globally search things generally speaking, so this setting is already a little bit weird. We probably need to get rid of that check here:

```csharp

        if (addExplorerInfo **&& isKeywordSearch** && !SettingsManager.Instance.HideExplorerSettingInfo)
        {
            resultsList.Add(GetExplorerInfoResult(infoIcon));
        }
```